test/e2e-test/pages/reason-for-appeal/reason-for-appeal.ts

```diff
@@ -0,0 +1,27 @@
+module.exports = {
+  reasonsForAppeal(I) {
+    When(/^I visit reasons for appeal$/, async () => {
+      await I.amOnPage('https://localhost:3000/reasons/reason-for-appeal');
+    });
+    Then(/^I enter "([^"]*)" into the reason for appeal text box and click Save and Continue$/, async (text: string) => {
+      await I.fillField('#moreDetail',text);
+      await I.click('.govuk-button');
```

Comment:
.govuk-button is too generic, every button would have that class.

You instead can use I.click('Save and continue') or 'Save for later'.